src/seedu/addressbook/AddressBook.java

```diff
@@ -217,13 +235,22 @@ public static void main(String[] args) {
      * signature anyway.
      * ====================================================================
      */
+	
+	/**
+	 * Displays a standard welcome message to user
+	 */
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.